Expanded explanation of where the tenant name is used.

michaelblumenthal opened the pull request:
I added content to better explain where the value in the tenant name field is used across many parts of Microsoft 365.

michaelblumenthal committed:
Interim commit

started working on the Tenant Name change updates

michaelblumenthal committed:
Update change-address-contact-and-more.md - fixed bullet levels

Fixed the bulleting so nesting showed correctly

michaelblumenthal committed:
Update change-address-contact-and-more.md

michaelblumenthal committed:
Added link from Name row in table to section below table

Linked from Name's description text to the section about Name changes.

michaelblumenthal committed:
Indenting fixed

michaelblumenthal committed:
Update change-address-contact-and-more.md name section

michaelblumenthal committed:
Update change-address-contact-and-more.md some more

michaelblumenthal committed:
Update change-address-contact-and-more.md

Fixed copy paste error about SharePoint path

michaelblumenthal committed:
Update change-address-contact-and-more.md

michaelblumenthal committed:
Update change-address-contact-and-more.md

michaelblumenthal committed:
Update change-address-contact-and-more.md

michaelblumenthal committed:
Update change-address-contact-and-more.md

michaelblumenthal committed:
Indentation fix, MS Teams sectio

michaelblumenthal committed:
Restructure a sentence for clarity

michaelblumenthal committed:
Fixed a sentence that was missing a word

michaelblumenthal commented:
@microsoft-github-policy-service agree

Michael Blumenthal
Microsoft MVP –M365 Apps and Services

Blog: https://MichaelBlumenthal.me
LinkedIn: http://bit.ly/MBB-LI
Email: ***@***.*** / ***@***.***
https://m365chicago.com
________________________________
From: learn-build-service-prod[bot] ***@***.***>
Sent: Wednesday, July 5, 2023 6:02:03 PM
To: MicrosoftDocs/microsoft-365-docs ***@***.***>
Cc: Michael Blumenthal ***@***.***>; Mention ***@***.***>
Subject: Re: [MicrosoftDocs/microsoft-365-docs] Expanded explanation of where the tenant name is used. (PR #12392)


Learn Build status updates of commit f6876e2<https://github.com/michaelblumenthal/microsoft-365-docs/commits/f6876e2821435282aab24b119f0acb2077d050d4>:

⚠️ Validation status: warnings
File    Status  Preview URL     Details
microsoft-365/admin/manage/change-address-contact-and-more.md<https://github.com/michaelblumenthal/microsoft-365-docs/blob/public/microsoft-365/admin/manage/change-address-contact-and-more.md>        ⚠️Warning               Details

microsoft-365/admin/manage/change-address-contact-and-more.md<https://github.com/michaelblumenthal/microsoft-365-docs/blob/public/microsoft-365/admin/manage/change-address-contact-and-more.md>

  *   Line 104, Column 1: [Warning: hard-coded-locale - See documentation<https://review.learn.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main>] Link 'https://learn.microsoft.com/en-us/office/troubleshoot/activation/reset-office-365-proplus-activation-state' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.
  *   Line 104, Column 1: [Suggestion: docs-link-absolute - See documentation<https://review.learn.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main>] Absolute link 'https://learn.microsoft.com/en-us/office/troubleshoot/activation/reset-office-365-proplus-activation-state' will be broken in isolated environments. Replace with a relative link.

For more details, please refer to the build report<https://buildapi.docs.microsoft.com/Output/PullRequest/76d60323-5511-5963-adf9-59162c800b4a/202307052258248548-12392/BuildReport?accessString=a7d977bfc4a550d5518cc8826144e58fb2336f3df0ce35c005408261ceaf9155>.

Note: Your PR may contain errors or warnings or suggestions unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these instructions<https://review.learn.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch> to resolve them.

For any questions, please:

  *   Try searching the learn.microsoft.com contributor guides<https://review.learn.microsoft.com/en-us/help/?branch=main>
  *   Post your question in the Learn support channel<https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47>

—
Reply to this email directly, view it on GitHub<https://github.com/MicrosoftDocs/microsoft-365-docs/pull/12392#issuecomment-1622648530>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ADTZSR4TAUAUVDY22HFYSHDXOXW6XANCNFSM6AAAAAAZ7SIQIM>.
You are receiving this because you were mentioned.Message ID: ***@***.***>

yogkumgit commented:
@JohanFreelancer9

michaelblumenthal commented:
At this point, is there anything more for me to do or will cmcatee and yogkumgit handle it from here?  Who needs to resolve the two items (a warning and a suggestion) about a URL I included?

yogkumgit commented:
@michaelblumenthal Please resolve the warning and suggestion. . If you see any copy review comment from John you have to fix that also then Cern McAtee will merge it.

yogkumgit commented:
@JohanFreelancer9

michaelblumenthal committed:
Merge branch 'MicrosoftDocs:public' into public

michaelblumenthal committed:
consistency and compliance updates in 

Changed "tenant" name to "organization" name for consistency and make all Learn.microsoft.com URLs relative.  Took out the en-us fragment from a support.office.com URL.

michaelblumenthal committed:
Update change-address-contact-and-more.md

Changed a use of "organization" back to "tenant" for clarity.

michaelblumenthal committed:
Merge branch 'public' into public

JohanFreelancer9 commented:
@yogkumgit, I'll do a copy review if required once the author has reviewed and approved this addition. Many thanks.

michaelblumenthal committed:
Merge branch 'public' into public

michaelblumenthal commented:
I fixed the warning about absolute URLs and hardcoded /en-us/. But now it doesn't like my relative path. What is the correct relative path to https://learn.microsoft.com/en-us/office/troubleshoot/activation/reset-office-365-proplus-activation-state ?

JohanFreelancer9 commented:
> I fixed the warning about absolute URLs and hardcoded /en-us/. But now it doesn't like my relative path. What is the correct relative path to https://learn.microsoft.com/en-us/office/troubleshoot/activation/reset-office-365-proplus-activation-state ?

/office/troubleshoot/activation/reset-office-365-proplus-activation-state

michaelblumenthal committed:
Update change-address-contact-and-more.md - fixed relative URL on line 104

Fixed the relative URL on line 104.

michaelblumenthal committed:
Merge branch 'public' into public

michaelblumenthal commented:
Ok, so the URLs are fixed. Do I need to do anything else?

yogkumgit commented:
@JohanFreelancer9

JohanFreelancer9 commented on microsoft-365/admin/manage/change-address-contact-and-more.md:
```diff
@@ -63,13 +63,49 @@ An explanation of each field is provided below.
 |**Field**  |**Description**  |
 |---------|---------|
-|Name  <br/>   | The name entered here is what users will see on the following pages:  <br/>  Sign-in page: If your users have set up other Microsoft accounts with their business or school email address, they may see the organization name on the sign-in page. This helps them distinguish between their work or school account and their other accounts, so they can identify which one to use when they  ...
+|Name  <br/>   | The name entered here is the organization's name. [See below for more information on where this is used](./change-address-contact-and-more.md#change-the-organizations-name).  <br/>  |
```
```suggestion
|Name  <br/>   | The name entered here is the organization's name. See [Change the organization's name](./change-address-contact-and-more.md#change-the-organizations-name) for more information on where this is used.  <br/>  |
```

JohanFreelancer9 commented on microsoft-365/admin/manage/change-address-contact-and-more.md:
```diff
@@ -63,13 +63,49 @@ An explanation of each field is provided below.
 |Technical contact  <br/> |This is the email address for the primary technical person who administers your Microsoft 365 subscription. This is the person who will receive communications about Microsoft 365 service status.  <br/> |
 |Preferred language  <br/> |The preferred language determines the language for all communications that are sent from Microsoft to your organization. When you sign up, this setting determines the language used by SharePoint Online, which your users see on your team site. If you change the language preference setting after you sign up, all future communications are sent in the most recent language  ...
 
+## Change the organization's Name
```
```suggestion
## Change the organization's name

```

JohanFreelancer9 commented on microsoft-365/admin/manage/change-address-contact-and-more.md:
```diff
@@ -63,13 +63,49 @@ An explanation of each field is provided below.
 
+## Change the organization's Name
+The organization's name is used throughout Microsoft 365, including but not limited to:  
+* Azure Active Directory Sign-in dialogs and multi-factor authentication prompts.  This includes multifactor authentication prompts provided by the Microsoft Authenticator app on iOS and Android devices. 
```
```suggestion
* Azure Active Directory Sign-in dialogs and multifactor authentication prompts. This includes multifactor authentication prompts provided by the Microsoft Authenticator app on iOS and Android devices. 
```

JohanFreelancer9 commented on microsoft-365/admin/manage/change-address-contact-and-more.md:
```diff
@@ -63,13 +63,49 @@ An explanation of each field is provided below.
+  * In Viva Engage, the organization name is used as the name of the home Engage network, and this shows up in the left navigation.  
+* OneDrive Sync
+  * The organization name is shown in File Explorer on Windows and Finder on Mac. 
+    * On Windows, in File Explorer, in the navigation pane, the blue cloud icon for OneDrive for your M365 tenant is labeled "OneDrive - \<organization name\>" (for example, "OneDrive - Contoso").  
```
```suggestion
    * On Windows, in File Explorer, in the navigation pane, the blue cloud icon for OneDrive for your Microsoft 365 tenant is labeled "OneDrive - \<organization name\>" (for example, "OneDrive - Contoso").  
```

JohanFreelancer9 commented on microsoft-365/admin/manage/change-address-contact-and-more.md:
```diff
@@ -63,13 +63,49 @@ An explanation of each field is provided below.
+    * The file path for the Pictures Known Folder is C:\users\<username>\OneDrive - \<organization Name>\Pictures
+    * The file path for the Desktop Known Folder is C:\users\<username>\OneDrive - \<organization Name>\Desktop
+  * The organization name is shown in:
+    * the OneDrive activity center
```
```suggestion
    * The OneDrive activity center
```

JohanFreelancer9 commented on microsoft-365/admin/manage/change-address-contact-and-more.md:
```diff
@@ -63,13 +63,49 @@ An explanation of each field is provided below.
+  * The organization name is shown in:
+    * the OneDrive activity center
+    * The tooltip of the OneDrive cloud icon
+    * the OneDrive settings window, on the Accounts tab. 
```
```suggestion
    * the OneDrive settings window, on the Accounts tab
```

JohanFreelancer9 commented on microsoft-365/admin/manage/change-address-contact-and-more.md:
```diff
@@ -63,13 +63,49 @@ An explanation of each field is provided below.
+    * The tooltip of the OneDrive cloud icon
+    * the OneDrive settings window, on the Accounts tab. 
+  *  Currently, updating the organization name does not update it for configured clients. 
+*  Microsoft Teams:
+    * Tenant Switcher in Teams displays the organization Name when a user participates in Teams in more than one tenant.
+    * Organizations that are under a certain size (get an organization-wide Team created automatically)[../../microsoftteams/create-an-org-wide-team].  That team gets its name from the organization's Name, but the team name does not change if the organization's Name field value changes.
```
```suggestion
*  Microsoft Teams
    * Tenant Switcher in Teams displays the organization name when a user participates in Teams in more than one tenant.
    * Organizations that are under a certain size (get an organization-wide team created automatically)[../../microsoftteams/create-an-org-wide-team]. That team gets its name from the organization's name, but the team name does not change if the organization's Name field value changes.
```

JohanFreelancer9 commented on microsoft-365/admin/manage/change-address-contact-and-more.md:
```diff
@@ -63,13 +63,49 @@ An explanation of each field is provided below.
+    * Organizations that are under a certain size (get an organization-wide Team created automatically)[../../microsoftteams/create-an-org-wide-team].  That team gets its name from the organization's Name, but the team name does not change if the organization's Name field value changes.
+*  OneNote:  End users may have a notebook named in this pattern: \<first name>@\<organization name>.
+*  Windows desktop applications for Word, Excel, and PowerPoint: The File Save and File Open screens display the organization name.  
+*  Mobile apps such as Word, Excel, PowerPoint, OneNote, Outlook, and the M365 App may display the organization name on various screens, especially ones that let you pick files from M365 services such as SharePoint and OneDrive.
```
```suggestion
*  Mobile apps such as Word, Excel, PowerPoint, OneNote, Outlook, and the Microsoft 365 App may display the organization name on various screens, especially ones that let you pick files from Microsoft 365 services such as SharePoint and OneDrive.
```

JohanFreelancer9 commented on microsoft-365/admin/manage/change-address-contact-and-more.md:
```diff
@@ -63,13 +63,49 @@ An explanation of each field is provided below.
+*  OneNote:  End users may have a notebook named in this pattern: \<first name>@\<organization name>.
+*  Windows desktop applications for Word, Excel, and PowerPoint: The File Save and File Open screens display the organization name.  
+*  Mobile apps such as Word, Excel, PowerPoint, OneNote, Outlook, and the M365 App may display the organization name on various screens, especially ones that let you pick files from M365 services such as SharePoint and OneDrive.
+*  Power Platform - the default environment takes its name from the organization name. The organization name (refered to as the TenantName in this article)[../../power-platform/guidance/adoption/secure-default-environment#rename-the-default-environment] is the default environment name, but  can be changed, though that is a manual step after a tenant rename.
```
```suggestion
*  Power Platform: The default environment takes its name from the organization name. The organization name (referred to as the TenantName in [this article](../../power-platform/guidance/adoption/secure-default-environment#rename-the-default-environment)) is the default environment name, but can be changed, though that is a manual step after a tenant rename.
```

JohanFreelancer9 commented on microsoft-365/admin/manage/change-address-contact-and-more.md:
```diff
@@ -63,13 +63,49 @@ An explanation of each field is provided below.
+*  Mobile apps such as Word, Excel, PowerPoint, OneNote, Outlook, and the M365 App may display the organization name on various screens, especially ones that let you pick files from M365 services such as SharePoint and OneDrive.
+*  Power Platform - the default environment takes its name from the organization name. The organization name (refered to as the TenantName in this article)[../../power-platform/guidance/adoption/secure-default-environment#rename-the-default-environment] is the default environment name, but  can be changed, though that is a manual step after a tenant rename.
+
+Note that only Microsoft Teams will automatically detect the change to the organization name and reflect the change in the tenant switcher.  Any teams, such as an organization-wide team named with the company name, will not update.  
+None of the other applications pick up the name change automatically.  Mobile apps may require you to sign out and sign back in or reinstall them before they will display the new name. 
```
```suggestion
Note that only Microsoft Teams will automatically detect the change to the organization name and reflect the change in the tenant switcher. Any teams, such as an organization-wide team named with the company name, will not update. 
None of the other applications pick up the name change automatically. Mobile apps may require you to sign out and sign back in or reinstall them before they will display the new name. 
```

JohanFreelancer9 commented on microsoft-365/admin/manage/change-address-contact-and-more.md:
```diff
@@ -63,13 +63,49 @@ An explanation of each field is provided below.
+
+Note that only Microsoft Teams will automatically detect the change to the organization name and reflect the change in the tenant switcher.  Any teams, such as an organization-wide team named with the company name, will not update.  
+None of the other applications pick up the name change automatically.  Mobile apps may require you to sign out and sign back in or reinstall them before they will display the new name. 
+[There is a script](/office/troubleshoot/activation/reset-office-365-proplus-activation-state) that can help you force Office desktop applications to pick up the new name, but it forces the user to sign back in to the Ofice desktop apps again.
```
```suggestion
[This script](/office/troubleshoot/activation/reset-office-365-proplus-activation-state) can help you force Office desktop applications to pick up the new name, but it forces the user to sign back into the Ofice desktop apps again.
```

JohanFreelancer9 commented on microsoft-365/admin/manage/change-address-contact-and-more.md:
```diff
@@ -63,13 +63,49 @@ An explanation of each field is provided below.
+None of the other applications pick up the name change automatically.  Mobile apps may require you to sign out and sign back in or reinstall them before they will display the new name. 
+[There is a script](/office/troubleshoot/activation/reset-office-365-proplus-activation-state) that can help you force Office desktop applications to pick up the new name, but it forces the user to sign back in to the Ofice desktop apps again.
+
+If you change the organization name, OneDrive Sync will not automatically update the organization name it uses.  The OneDrive Sync client will not notice the new name unless you [unlink OneDrive Sync client and re-link it](https://support.microsoft.com/office/unlink-and-re-link-onedrive-3c4680bf-cc36-4204-9ca5-e7b24cdd23ea) by signing in again. Note that you may need to do some file system cleanu ...
+
```
```suggestion
If you change the organization name, OneDrive Sync will not automatically update the organization name it uses. The OneDrive Sync client will not notice the new name unless you [unlink OneDrive Sync client and re-link it](https://support.microsoft.com/office/unlink-and-re-link-onedrive-3c4680bf-cc36-4204-9ca5-e7b24cdd23ea) by signing in again. Note that you may need to do some file system cleanup after that. Also, you don't want to unlink your OneDrive Sync client if you have unsynced files. Make sure syncing has been completed before unlinking. Relinking will resync your OneDrive, but if you had previously synced SharePoint libraries that are listed under the blue office building in File Explorer, you would need to re-sync those libraries individually. A better alternative is to use the Add to OneDrive feature to add a shortcut in your OneDrive to a SharePoint library. With an Add to OneDrive shortcut, the folder contents syncs within your OneDrive blue cloud instead of your OneDrive blue office building and would automatically be included in syncing when relinking.
```

JohanFreelancer9 reviewed (changes requested):
Copy review - @michaelblumenthal, kindly commit the suggested changes. Many thanks.

michaelblumenthal committed:
Update microsoft-365/admin/manage/change-address-contact-and-more.md

Co-authored-by: JohanFreelancer9 <48568725+JohanFreelancer9@users.noreply.github.com>

michaelblumenthal committed:
Update microsoft-365/admin/manage/change-address-contact-and-more.md

Co-authored-by: JohanFreelancer9 <48568725+JohanFreelancer9@users.noreply.github.com>

michaelblumenthal committed:
Update microsoft-365/admin/manage/change-address-contact-and-more.md

Co-authored-by: JohanFreelancer9 <48568725+JohanFreelancer9@users.noreply.github.com>

michaelblumenthal committed:
Update microsoft-365/admin/manage/change-address-contact-and-more.md

Co-authored-by: JohanFreelancer9 <48568725+JohanFreelancer9@users.noreply.github.com>

michaelblumenthal committed:
Update microsoft-365/admin/manage/change-address-contact-and-more.md

Co-authored-by: JohanFreelancer9 <48568725+JohanFreelancer9@users.noreply.github.com>

michaelblumenthal committed:
Update microsoft-365/admin/manage/change-address-contact-and-more.md

Co-authored-by: JohanFreelancer9 <48568725+JohanFreelancer9@users.noreply.github.com>

michaelblumenthal committed:
Update microsoft-365/admin/manage/change-address-contact-and-more.md

Co-authored-by: JohanFreelancer9 <48568725+JohanFreelancer9@users.noreply.github.com>

michaelblumenthal committed:
Update microsoft-365/admin/manage/change-address-contact-and-more.md

Co-authored-by: JohanFreelancer9 <48568725+JohanFreelancer9@users.noreply.github.com>

michaelblumenthal committed:
Update microsoft-365/admin/manage/change-address-contact-and-more.md

Co-authored-by: JohanFreelancer9 <48568725+JohanFreelancer9@users.noreply.github.com>

michaelblumenthal committed:
Update microsoft-365/admin/manage/change-address-contact-and-more.md

Co-authored-by: JohanFreelancer9 <48568725+JohanFreelancer9@users.noreply.github.com>

michaelblumenthal committed:
Update microsoft-365/admin/manage/change-address-contact-and-more.md

Co-authored-by: JohanFreelancer9 <48568725+JohanFreelancer9@users.noreply.github.com>

michaelblumenthal committed:
Update microsoft-365/admin/manage/change-address-contact-and-more.md

Co-authored-by: JohanFreelancer9 <48568725+JohanFreelancer9@users.noreply.github.com>

michaelblumenthal committed:
Merge branch 'public' into public

michaelblumenthal committed:
Merge branch 'public' into public

michaelblumenthal committed:
Update change-address-contact-and-more.md - corrected relative URL for link to Power Platform page

Changed ../../power to /power

michaelblumenthal commented:
I accepted all of @JohanFreelancer9 revisions (Thank you @JohanFreelancer9!). I fixed another relative url issue that the system flagged.  Let me know if I need to do anything else.

yogkumgit commented:
@cmcatee-MSFT Please merge this PR. Thanks

michaelblumenthal committed:
Merge branch 'public' into public

michaelblumenthal committed:
Merge branch 'public' into public

michaelblumenthal committed:
Merge branch 'public' into public

michaelblumenthal committed:
Merge branch 'public' into public

michaelblumenthal commented:
When will this PR get merged? Should I still be updating the branch to keep it in sync?

michaelblumenthal committed:
Merge branch 'public' into public

cmcatee-MSFT commented:
@michaelblumenthal This article is currently undergoing a revision. I've pulled your suggestion into the working draft, which I'll be reviewing with the PM shortly. As soon as he approves the changes, I'll be able to republish it. Thanks for your submission.

cmcatee-MSFT commented:
@michaelblumenthal Would you say a bit more about why this information is needed, particularly in this article?

michaelblumenthal commented:
This information is needed because IT Admins need to understand the impact of changing the tenant display name.  Even though the act of changing the display name is simple, the repercussions of doing so are far reaching and not obvious.   Do you think this topic deserves its own page?

michaelblumenthal commented:
@cmcatee-MSFT any update on when the revision will be published?

michaelblumenthal commented:
@cmcatee-MSFT , was my content ever used somewhere? I don't see it in the main article.

cmcatee-MSFT commented:
@michaelblumenthal I discussed this with the PM and then forgot to reply here, I apologize for that. The PM didn't think we needed to include this level of detail in this article, so the proposed change was rejected.

michaelblumenthal commented:
@cmcatee-MSFT thank you for letting me know. I can understand that as an editorial decision about what belongs on this page, however I believe my content about where the name is used is still highly valuable to tenant admins considering changing the name. Could my content be published on a new page?
